code/game/machinery/computer/law.dm

```diff
@@ -16,6 +23,11 @@
 		if(!current)
 			to_chat(user, "<span class='caution'>You haven't selected anything to transmit laws to!</span>")
 			return
+		var/input = stripped_input(user, "Please enter the Upload code.", "Uplode Code Check")
+		if(!GLOB.upload_code)
+			GLOB.upload_code = random_code(4)
+		if(!(input == GLOB.upload_code))
```

Comment:
```suggestion
		if(input != GLOB.upload_code)
```

Reply:
Also needs an error message for wrong code.

Reply:
Done